src/json-completion.ts

```diff
@@ -333,7 +371,7 @@ export class JSONCompletion {
         }
       }
     }
-    if (!value || nValueProposals > 1) {
+    if (!value || nValueProposals < 1) {
```

Comment:
I wasn't sure what the intention was here, so I reversed the logic

in the case of `packageJson.types` we had two value proposals - one from `default`, and another because it was an enum

Reply:
I think the idea is that if there are multiple possible values, we don't assume a value for the user

Reply:
I think in this case, a `schema` has both `default` and `enum`. in my opinion, we should just guess with `default` and forego the other snippet guessing logic then, instead of counting? what do you think?

Reply:
@imolorhe I have pushed another commit here where i reverted the logic for more than 1 proposal, but made it so that if default is provided, only the default value is proposed, how does that sound?